helpers/constants.ts

```diff
@@ -9,3 +9,9 @@ export const ZERO_BD = BigDecimal.zero();
 export const ONE_BD = BigDecimal.fromString("1");
 export const ZERO_ADDRESS = Address.fromString("0x0000000000000000000000000000000000000000");
 export const ONE_ETHER_IN_WEI = BigInt.fromI32(10).pow(18);
+export const ONE_DAY_BI = BigInt.fromI32(86400);
+
+// TODO: Update it with the real address
+export const LOOKSRARE_AGGREGATOR = Address.fromString("0x0000000000000000000000000000000000069420");
+export const LOOKSRARE_AGGREGATOR_SWEEP_EVENT_TOPIC =
```

Comment:
Maybe it would be possible to derive a helper for calculating it?

Reply:
Updated